### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by caiyongqi** [Xharlie/pointnerf#19]
In base_rendering_model, `loss = self.l2loss(masked_output, masked_gt) * masked_gt.shape[1]`, why multiply `masked_gt.shape[1]`?
https://github.com/Xharlie/pointnerf/blob/a614e1d15cc8409e14f80c7b3dd0091939fef758/models/base_rendering_model.py#L560

**Comment by Xharlie** [comment]
hi this loss is not used for gradient update, it is to indicate the ray miss, so it is convenient to look at if it's not super small. Its loss weight is always zero, i suggest you print out the loss weight to confirm.

**Comment by caiyongqi** [comment]
What does it mean that the loss weight is not zero? The loss weight is not zero when fine-tuning on own dataset. Thanks!

**Comment by Xharlie** [comment]
it is zero, thanks!
